modules/storage-persistent-storage-efs-pvc.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-efs.adoc
+
+[id="efs-pvc_{context}"]
+= Creating the persistent volume claim
+
+.Prerequisites
+
+* The EFS provisioner pod has been created successfully.
+
+.Procedure
+
+. In the {product-title} console, click *Storage* -> *Persistent Volume Claims*.
+
+. In the persistent volume claims overview, click *Create Persistent Volume
+Claim*.
+
+. Define the desired options on the page that appears.
```

Comment:
Not "desired."
> :no_entry_sign: desired adj
Do not use. Use “required” or “the <item> that you want.”

Probably not "appears."

> ! appear v
Do not use unless there is no better alternative. Possible alternatives include “is displayed” and “opens.”